### PR TITLE
added code convention

### PR DESCRIPTION
**Pull request by nik00did**
Check pull request and try to understand what I did.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
\ No newline at end of file
+# Calculator For Me
+Code convention(code style)
+--------
+```
+HTML
+```
+1. Все стили подключаются через `<link/>` в `<head></head>`.
+2. Путь пишем относительный.
+3. Не ставим enters для пукстых строк в html коде.
```
Comment (creature17) [inline README.md]: не пукай

File: README.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+4. Если в теге стоит только тексте, меньше 120 символов, то не переносим закрывающий тег на новую строку.
```
Comment (creature17) [inline README.md]: орфографист чертов

File: README.md [hunk 3]
```diff
@@ -1 +1,58 @@
...
+5. Если два и больше елемента в теге то закрывающий тег и сождержимое переносим на новую строку.
```
Comment (creature17) [inline README.md]: граммотей блин

File: README.md (hunk 1 above)
Comment (nik00did) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (nik00did) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (nik00did) [reply README.md]: done

**Review by creature17 — APPROVED** [review]

**Review by creature17 — APPROVED** [review]

**Pull request merged**